_pkgdown.yml

```diff
@@ -21,27 +21,29 @@ reference:
 
 - title: Individual linters
   contents:
-  - all_undesirable_functions
   - ends_with("linter")
 
-- title: Customer linters
+- title: Lists of defaults
+  contents:
+  - all_undesirable_functions
+  - starts_with("default_")
+
+- title: Utilities
   contents:
   - Linter
-  - expect_lint
-  - expect_lint_free
+  - starts_with("expect_")
```

Comment:
this will include a bunch of linters, right? `expect_identical_linter()` and friends...

Reply:
Ah, yes. I have changed it to `starts_with("expect_lint")`, which should avoid this and select the only two we have in mind here:

``` r
library(tidyselect)

starts_with(
  "expect_lint",
  vars = c("expect_lint", "expect_lint_free", "expect_identical_linter")
)
#> [1] 1 2
```

<sup>Created on 2022-07-31 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

Reply:
given that there are only two, I'd rather just write them out. it's better to be explicit unless it's really saving us a lot of typing/maintenance/upkeep

Reply:
Okay, sounds reasonable. I've reverted this change.